monai/transforms/utils.py

```diff
@@ -1496,5 +1498,20 @@ def convert_pad_mode(dst: NdarrayOrTensor, mode: Union[NumpyPadMode, PytorchPadM
     raise ValueError(f"unsupported data type: {type(dst)}.")
 
 
+def as_contiguous(data):
```

Comment:
I put this utility in the `transforms` folder because I think we may consider to add a `AsContiguous` transform later.

Thanks.